### DISCUSSION TITLE
Remove individual actions from installed packs

### DISCUSSION THREAD

**Discussion by lexiismadd** [StackStorm/st2#6119] [Packs]
Hi,
I've installed an in-house pack, and over time have removed actions from the source, but there are still legacy actions installed in Stackstorm.  How can I remove those legacy actions without completely uninstalling and reinstalling the pack?

**Comment by nzlosh** [comment]
Use the delete functionality https://docs.stackstorm.com/actions.html#modification-in-action-delete-api
